src/binary/ibc-setup/utils/resolve-option.ts

```diff
@@ -1,14 +1,13 @@
 export function resolveOption(
-  identifier: string,
-  ...args: Array<(string | undefined) | (() => string | undefined | void)>
+  ...args: Array<(string | undefined) | (() => string | undefined)>
```

Comment:
It turned out that `void` adds unnecessary complexity to handling. I removed it as you requested before. @ethanfrey